config/environment.js

```diff
@@ -2,5 +2,11 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+    moment: {
+      // To cherry-pick specific locale support into your application.
+      // Full list of locales: https://github.com/moment/moment/tree/2.10.3/locale
+      includeLocales: ['en', 'fr']
```

Comment:
I think that if you want to merge back your changes in the main repo, you should only include `en` locale.